dashboard/app/views/levels/_level_group.haml

```diff
@@ -39,7 +39,7 @@
 
     .level-group-number= "#{page.offset + index + 1}. "
     .level-group-content
-      - if level_class == "multi"
+      - if ['multi', 'evaluation_question'].include? level_class
```

Comment:
Yeah, I think evaluation_multi would capture the similarities but differences more here.